src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,45 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin onTablet {
```

Comment:
It's a good practice to use variables for breakpoints in media queries. Consider defining a variable for the tablet breakpoint (e.g., $tablet-breakpoint: 744px) and use it in the onTablet mixin.